src/Geometry/axistensors.jl

```diff
@@ -244,6 +244,9 @@ AxisVector(ax::A1, v::SVector{N, T}) where {A1 <: AbstractAxis, N, T} =
     arg3::Real,
 ) where {A} = AxisVector(A.instance, SVector(arg1, arg2, arg3))
 
+(AxisVector{T, A, SVector{0, T}})() where {A, T} =
```

Comment:
```suggestion
(AxisVector{T, A, SVector{0, T}} where {T})() where {A} =
```
The previous 3 methods all have this type signature. Not sure if it matters but probably best to be consistent.